lecture_1/lecture_1_passed/passed.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/sethgrid/pester"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+type passedAPIResponse struct{
+	Name string
+	Age int
+	Passed bool
+	Skills []string
+}
+
+
+
+const passedUrl = "https://run.mocky.io/v3/f7ceece5-47ee-4955-b974-438982267dc8"
+
+func linearBackoff(retry int) time.Duration {
+	return time.Duration(retry) * time.Second
+}
+
+func writingToFile(f *os.File, decodedContent []passedAPIResponse){
+	for i := range decodedContent{
```

Comment:
```suggestion
	for _, applicant := range decodedContent{
```

And then use `applicant` directly instead of using index to dereference the array.

Also - as @sodic already pointed out - `decodedContent` is not really a descriptive name here.